spelling_wordlist.txt

```diff
@@ -319,12 +366,15 @@ virtualrunenv
 Virtualrunenv
 vswhere
 watchOS
+weren
```

Comment:
weren??

Reply:
Words like `weren't` are not well recognized